testing/dart/platform_dispatcher_test.dart

```diff
@@ -39,4 +44,60 @@ void main() {
       return ViewConfiguration(view: view)..copyWith(view: view, window: view);
     });
   });
+
+  test("PlatformDispatcher and PlatformConfiguration have matching getters", () {
+      final List<Pattern> exclusionCriteria = <Pattern>[
+        RegExp(r'^_.*'), // exclude any private members
+        RegExp(r'^on[A-Z].*'), // exclude any callbacks
+        RegExp(r'locale$'), // locale is a convenience getter for interacting with `locales` which _is_ backed by the config
+        'instance', // exclude the static instance of PlatformDispatcher
+        'configuration', // the configuration should not reference itself
+        'views', // views are backed by their own config and don't need to be referenced in the platform config
+        'initialLifecycleState', // default route is stored/retrieved from the platform, not the engine
+        'frameData', // framed data updates too often and would kick off too many config changes
+      ];
```

Comment:
@loic-sharma could you give me a spot-check on the justifications for these exclusions? Specifically `initialLifecycleState` and `frameData`. I'm not sure on the reasons why those properties _weren't_ backed by the config, but their implementations were different enough that I expected they _were_ intended to be different.

Reply:
Never mind, had to remove this test due to the `analyzer` dependency.